misc/make-deps.sh

```diff
@@ -4,6 +4,11 @@ XPWD=`pwd`
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"	# dir!
 cd "${ROOT}" >/dev/null
 
+function errorhandler () {
+	echo "Something went wrong! Hey, on the bright side if it breaks, you can keep both halves!"
```

Comment:
As much as I love this message, can you write something more specific to what happened, possibly mentioning a dep script? You could even do

```install thing || errorhandler "some message"

and use `"$1"` in the error handler for more info :)